src/mvPyUtils.cpp

```diff
@@ -2396,6 +2396,12 @@ GenerateCoreFile(std::ofstream& stream)
                 stream << "\n\t\twarnings.warn('" << args.name << " keyword renamed to " << args.new_name << "', DeprecationWarning, 2)";
                 stream << "\n\t\t" << args.new_name << "=kwargs['" << args.name << "']";
             }
+
+            else if (args.arg_type == mvArgType::DEPRECATED_KEYWORD_ARG)
+            {
+                stream << "\n\n\tif '" << args.name << "' in kwargs.keys():";
+                stream << "\n\t\twarnings.warn('" << args.name << " keyword deprecated. See " << args.new_name << " argument', DeprecationWarning, 2)";
```

Comment:
I think we need more flexibility on what we put after the words "keyword deprecated". It should be just a free-form message. Like, for some arguments we want to deprecate them without any extra info; for others we're providing an alternative way to achieve that but instead of being a single argument, it might be a combination of values or a special technique.

We can still utilize `args.new_name` but treat it as a message, without surrounding it with "see" and "argument" (and the message should be optional).